README.md

```diff
@@ -30,10 +32,32 @@ Note that the version on the webstore may not be as up-to-date as on GitHub.
 
 ## How to Use
 
-1. Press Ctrl + Shift + Space to cycle through the enabled keyboard languages.
-2. Type away…
-3. Ctrl + Space to toggle between the most recently used keyboard languages.
+After enabling multiple keyboard layouts an icon will appear in the system tray
```

Comment:
Link the Chromebook docs here? https://support.google.com/chromebook/answer/1059492